tls_codec/derive/tests/decode.rs

```diff
@@ -58,8 +58,10 @@ fn tuple_struct() {
     let serialized_t1 = t1.tls_serialize_detached().unwrap();
     let deserialized_t1 = TupleStruct1::tls_deserialize(&mut serialized_t1.as_slice()).unwrap();
     let (deserialized_bytes_t1, _remainder) =
-        <TupleStruct1 as tls_codec::DeserializeBytes>::tls_deserialize(serialized_t1.as_slice())
-            .unwrap();
+        <TupleStruct1 as tls_codec::DeserializeBytes>::tls_deserialize_bytes(
+            serialized_t1.as_slice(),
+        )
+        .unwrap();
     assert_eq!(t1, deserialized_t1);
```

Comment:
Here are a few casts left in this file.

```suggestion
        TupleStruct1::tls_deserialize_bytes(serialized_t1.as_slice()).unwrap();
```